tests/test_dois.py

```diff
@@ -0,0 +1,79 @@
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+from invenio.testsuite import InvenioTestCase, make_test_suite, run_test_suite
+from dojson.contrib.marc21.utils import create_record
+from inspirehep.dojson.hep import hep
+from inspirehep.dojson.utils import strip_empty_values
+
+
+class DoisTests(InvenioTestCase):
+
+    def setUp(self):
+        self.snippet_single_doi = ('<record><datafield tag="024" ind1="7" ind2=" ">'
+                                   '<subfield code="2">DOI</subfield>'
+                                   '<subfield code="a">10.1088/0264-9381/31/24/245004</subfield>'
+                                   '</datafield></record>')
+
+        self.snippet_duplicate_doi = ('<record><datafield tag="024" ind1="7" ind2=" ">'
+                                      '<subfield code="2">DOI</subfield>'
+                                      '<subfield code="9">bibmatch</subfield>'
+                                      '<subfield code="a">10.1088/1475-7516/2015/03/044</subfield>'
+                                      '</datafield>'
+                                      '<datafield tag="024" ind1="7" ind2=" ">'
+                                      '<subfield code="2">DOI</subfield>'
+                                      '<subfield code="a">10.1088/1475-7516/2015/03/044</subfield>'
+                                      '</datafield></record>')
+
+        self.snippet_multiple_dois = ('<record><datafield tag="024" ind1="7" ind2=" ">'
+                                      '<subfield code="2">DOI</subfield>'
+                                      '<subfield code="a">10.1103/PhysRevD.89.072002</subfield>'
+                                      '</datafield>'
+                                      '<datafield tag="024" ind1="7" ind2=" ">'
+                                      '<subfield code="2">DOI</subfield>'
+                                      '<subfield code="9">bibmatch</subfield>'
+                                      '<subfield code="a">10.1103/PhysRevD.91.019903</subfield>'
+                                      '</datafield></record>')
+
+    def test_single_doi(self):
+        try:
+            x = create_record(self.snippet_single_doi)
+            strip_empty_values(hep.do(x))
+        except:
+            raise Exception('Single dois snippet error')
+
+    def test_duplicate_doi(self):
+        try:
+            x = create_record(self.snippet_duplicate_doi)
+            strip_empty_values(hep.do(x))
+        except:
+            raise Exception('Duplicate dois snippet error')
+
+    def test_multiple_dois(self):
+        try:
+            x = create_record(self.snippet_multiple_dois)
+            strip_empty_values(hep.do(x))
+        except:
+            raise Exception('Multple dois snippet error')
+
+
+TEST_SUITE = make_test_suite(DoisTests)
```

Comment:
This boilerplate here is not needed anymore. It can be removed, along with the associated imports.